paperspace/method.py

```diff
@@ -135,7 +135,7 @@ def method(category, method, params):
             data = params
             params = None
         r = requests.request(http_method, config.CONFIG_HOST + path,
-                             headers={'x-api-key': config.PAPERSPACE_API_KEY},
+                             headers={'x-api-key': config.PAPERSPACE_API_KEY, 'ps_client_name': 'paperspace-python', 'ps_client_version' : "0.0.14"},
```

Comment:
Ideally we would pull in a single `__version__` here so it doesn't have to be manually updated. Not sure the best way to do this.

Reply:
There is a __version__ in the init of the main package, maybe we could use that one

Reply:
and just for PEP8 matter - line length shouldn't be longer 120 characters, so maybe break into separate lines, like 
```                             
headers={'x-api-key': config.PAPERSPACE_API_KEY,
                  'ps_client_name': 'paperspace-python',
                  'ps_client_version': "0.0.14"},
```